slices/libatomic1.yaml

```
@@ -0,0 +1,10 @@
+package: libatomic1
+
+slices:
+  libs:
+    essential:
+      - libc6_libs
+      # Although gcc-12-base is a package dependency, it is not needed for this
+      # libs slice and is thus omitted.
+    contents:
+      /usr/lib/*-linux-*/libatomic.so.1*:
```

```suggestion
      /usr/lib/*-linux-*/libatomic.so.1*:
      
```
Could we add a new line to the end of these YAML files?